scaling/scaling.cfg

```diff
@@ -0,0 +1,5 @@
+[scaling]
```

Comment:
Is it a sample file? Could you add a comment at the top?

Reply:
comment added. This file is used for local testing.